agent/instrumentation/micrometer-1.0/src/test/java/MicrometerTest.java

```diff
@@ -27,8 +28,6 @@
 
 class MicrometerTest {
```

Comment:
the changes in this class seem worth keeping even though they didn't fix anything